halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/deploy/ApplyDeployCommand.java

```diff
@@ -84,16 +84,24 @@
   )
   List<String> excludeServiceNames = new ArrayList<>();
 
+  @Parameter(
+      names = "--delete-orphan-services",
```

Comment:
s/orphan/orphaned/

Reply:
I don't know if you want to change the variable names as well here, matters less than the user facing stuff.